packages/console-types/src/types/services.ts

```diff
@@ -835,6 +835,17 @@ export const customResource = {
       },
       required: ['kind', 'apiVersion'],
     },
+    runtime: {
+      type: 'object',
+      required: ['type'],
+      properties: {
+        type: { type: 'string' },
```

Comment:
Can we use an enum or a const to narrow the string type?

Reply:
done